docs/webpack-dev-server.config.js

```diff
@@ -23,7 +23,7 @@ var config = {
     },
     //Modules will be searched for in these directories
     modulesDirectories: [
-      "web_modules",
```

Comment:
Hum, with the default `modulesDirectories` yes. But I think that since we overide it, it's no longer true.